spec/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author: User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', post_counter: 0), title: 'Hello', text: 'This is my first post', likes_counter: 0, comments_counter: 0) }
+
+  before { subject.save }
+
+  it 'Title must not be blank' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'CommentsCounter must be an integer greater than or equal to zero.' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'LikesCounter must be an integer greater than or equal to zero.' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] I equally realized that most of the tests are failing as you can see in the screenshot below. Please let's kindly fix :wrench:  those issues.


![Screenshot from 2023-03-02 10-11-26](https://user-images.githubusercontent.com/63087491/222384317-b793d57a-41c0-41eb-ae4a-8e891a43bdef.png)